### PR TITLE
refactor(shell): Refactoring shell handling

### PR DESCRIPTION
**Pull request by danielsuguimoto**

### CODE REVIEW

File: cmd/kool_task.go
Comment (fabriciojs) [inline cmd/kool_task.go:24]: any reason to not call the this simply `shell` instead of `taskShell`?

**Review by fabriciojs — APPROVED** [review]
@danielsuguimoto great job! Now `shell.Shell` will enable a couple things we had on hold.

**Comment by fabriciojs** [comment]
@danielsuguimoto a small conflict appeared - please can you take care of it? then we can merge. (I guess the other PR's will need updataing as soon as this gets merged, right?)

File: cmd/kool_task.go
Comment (danielsuguimoto) [reply cmd/kool_task.go:24]: kool task has two shells...one capture the original service output....the other handles the terminal...

**Comment by danielsuguimoto** [comment]
> @danielsuguimoto a small conflict appeared - please can you take care of it? then we can merge. (I guess the other PR's will need updataing as soon as this gets merged, right?)

@fabriciojs done!

**Pull request merged**
